mava/systems/ff_ippo.py

```diff
@@ -464,16 +471,12 @@ def run_experiment(_config: DictConfig) -> None:
     )
 
     # Setup evaluator.
-    evaluator, absolute_metric_evaluator, (trained_params, eval_keys) = evaluator_setup(
-        eval_env=eval_env,
-        key_e=key_e,
-        network=actor_network,
-        params=learner_state.params.actor_params,
-        config=config,
-    )
+    # Broadcast trained params to cores and split keys for each core.
+    trained_params = unreplicate_batch_dim(learner_state.params.actor_params)
```

Comment:
I don't think it is necessary to have trained params initialized before the for loop as we will get the first needed trained_params in line 533

Reply:
The same for all the other systems